var/spack/repos/builtin/packages/minizip/package.py

```diff
@@ -21,3 +21,19 @@ class Minizip(AutotoolsPackage):
     depends_on('libtool', type='build')
     depends_on('m4', type='build')
     depends_on('zlib')
+
+    @run_before('autoreconf')
+    def build_minizip_binary(self):
+        print(self)
+        bash = which('bash')
+        bash('./configure')
```

Comment:
```suggestion
        configure()
```
Does this run the correct configure script?

Reply:
> Does this run the correct configure script?

Yes, It works well.